client/asset/btc/btc_test.go

```diff
@@ -6173,3 +6188,277 @@ func TestFindBond(t *testing.T) {
 		})
 	}
 }
+
+func TestIDUnknownTx(t *testing.T) {
+	type idResult struct {
+		txType    asset.TransactionType
+		recipient *string
+		bondInfo  *asset.BondTxInfo
+		amt       uint64
+		fee       uint64
+	}
+
+	type test struct {
+		name           string
+		ltr            *ListTransactionsResult
+		txB            dex.Bytes
+		segwit         bool
+		chainParams    *chaincfg.Params
+		ownedAddresses map[string]bool
+		ownsAddress    bool
+		exp            *idResult
+	}
+
+	h2b := func(h string) []byte {
+		b, _ := hex.DecodeString(h)
+		return b
+	}
+
+	floatPtr := func(f float64) *float64 {
+		return &f
+	}
+	stringPtr := func(s string) *string {
+		return &s
+	}
+
+	tests := []*test{
+		{
+			name: "btc swap",
+			ltr: &ListTransactionsResult{
+				Send: true,
+			},
+			txB: h2b("01000000000101d291fc06a2f14c984bd2be00d4b319ba3ffe8131dbdaf8e8089df76e18d7f6400100000000ffffffff0200e1f505000000002200205c1989ae196beb93b57d26f714382d8c8b149d548d64804b9b6629da01133da4bcb62542020000001600148bb93295660c3530110a4eb1f8bd4a6c54fcbe050247304402205aa4c40e6bb5f79f5d0520f4e2928a989b48721785cb128805a8dafa88b8a6e5022018a9dd5478144c95aad7b2210b1d768ed81fda750d59af03435fe167aba53f070121037c226414b8613aa39e611958397ee727a622b4619a3837f28abc14eee7e4162c00000000"),
+			exp: &idResult{
```

Comment:
No hex txs please. We have all kinds of helpers in this file like `makeRawTx`, `makeSwapContract`, `makeRPCVin`, etc and a bunch of helpers in dexbtc to help construct these transactions in a way that we can read.